client/admin/js/main.js

```diff
@@ -13,6 +13,28 @@ const HierarchicalNav = require('o-hierarchical-nav');
 const nav = document.querySelector('.o-hierarchical-nav');
 new HierarchicalNav(nav);
 
+const troubleURLS = [];
+
+function pointOutTroubleMakers(){
+	const activeLinks = Array.from(document.querySelectorAll('.screen-page'));
+
+	activeLinks.forEach(function(activeLink){
+
+		const link = activeLink.getAttribute('href');
+
+		debugger;
+
+		troubleURLS.forEach(function(url){
```

Comment:
We could replace this with the [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) method.

Example:
```
if (troubleURLS.includes(link)) {
  activeLink.setAttribute('data-troublsome-url', 'true');
}
```

I'm not 100% but I think we could remove the `if` statement and go with:
`activeLink.setAttribute('data-troublsome-url', troubleURLS.includes(link));`